workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/InviteMemberModal/InviteMemberModal.tsx

```diff
@@ -161,14 +162,19 @@ export function InviteMemberModal({
     await addExistentUserIntoAccount(selectedAccount.id, formFields.email);
     await addAccountRolesToUser(user);
 
+    const accountGroups = await getAccountGroup(selectedAccount.id);
+
+    const accountGroupERC = accountGroups && accountGroups[0]?.externalReferenceCode;
+
     await addAdditionalInfo({
       acceptInviteStatus: false,
+      accountGroupERC,
       accountName: selectedAccount.name,
       emailOfMember: formFields.email,
       inviteURL:
         Liferay.ThemeDisplay.getPortalURL() +
         '/c/login?redirect=' +
-        getSiteURL(),
+        getSiteURL() + '/loadingpagevalidation',
```

Comment:
Why this page is called `loadingpagevalidation` ? why not just `loading` or `hang-tight` (using raylife as example)

From the user perspective, the URL's should be intuitive

```suggestion
        getSiteURL() + '/loadingpagevalidation',
```